.github/CODEOWNERS

```
@@ -0,0 +1,5 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+# The java-samples-reviewers team is the default owner for samples changes
+samples/*                         @GoogleCloudPlatform/java-samples-reviewers
```

```suggestion
samples/                         @stephaniewang526 @GoogleCloudPlatform/java-samples-reviewers
```

1. `samples/*` > `samples/` = makes the ownership recursive

2. `@stephaniewang526` = declare yourself as an owner as well - any changes someone else wants to make to these samples should also be run by a DPE assigned to that product